.github/CONTRIBUTING.md

```diff
@@ -39,8 +47,23 @@ Please review our [style guide](STYLEGUIDE.md) to ensure the least amount of rew
 
 <!-- When a tool is selected for Static Analysis add a section here to document what we care about and the process to change the things about it that we don't like -->
 
+## SwiftLint
+
+We use [SwiftLint](https://github.com/realm/SwiftLint) in our pipeline to check adherence to the style guide. Just like the style guide, we strongly recommend you install SwiftLint during development to ensure the least amount of rework and changes in your pull request.
+
+You can install via Homebrew:
+
+`brew install swiftlint`
+
+via CocoaPods:
+
+`pod 'SwiftLint`
```

Comment:
Is this really the syntax? `pod 'SwiftLint`
With the leading `'`?

Reply:
nope! I missed the trailing one. got mixed into the markdown back ticks

Reply:
Looking at this again, it may not be applicable to have the pod directions here, because really this would be a line in a Podfile, and I don't think that's how most contributors would be installing. 🤔

Reply:
I think that's a good call out. I was asking @Tyler-Keith-Thompson and he mentioned this would be fine, but I would likely mention the brew instructions only myself. I say we change it to that.